<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="usesScrollButtons">
         <bool>false</bool>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="DefaultTab">
         <attribute name="title">
          <string>Default Tab</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Load Layout File</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="updateBox">
          <property name="title">
           <string>Update Type</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="radioLive">
               <property name="text">
                <string>Live Changes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioOnSubmit">
               <property name="text">
                <string>Change on Submit</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="submitButton">
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updateButton">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>444</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSerial_Port"/>
    <addaction name="actionDisplay_Serial_Data"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuLayout">
    <property name="title">
     <string>Layout</string>
    </property>
    <addaction name="actionLoad_Setup_File"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLayout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSerial_Port"/>
   <addaction name="actionDisplay_Serial_Data"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionLoad_Setup_File"/>
  </widget>
  <action name="actionSerial_Port">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/Serial.ico</normaloff>:/icons/icons/Serial.ico</iconset>
   </property>
   <property name="text">
    <string>Serial Port</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/Close.ico</normaloff>:/icons/icons/Close.ico</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/Disconnect.ico</normaloff>:/icons/icons/Disconnect.ico</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionDisplay_Serial_Data">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/SerialDisplay.ico</normaloff>:/icons/icons/SerialDisplay.ico</iconset>
   </property>
   <property name="text">
    <string>Display Serial Data</string>
   </property>
  </action>
  <action name="actionLoad_Setup_File">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/Layout.ico</normaloff>:/icons/icons/Layout.ico</iconset>
   </property>
   <property name="text">
    <string>Load Layout File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
